client/my-sites/plugins/plugin-information/index.jsx

```diff
@@ -195,6 +209,13 @@ export default React.createClass( {
 			'is-singlesite': !! this.props.siteVersion
 		} );
 
+		const plugin = this.props.selectedSite && this.props.sites[ 0 ] ? this.props.sites[ 0 ].plugin : this.props.plugin;
```

Comment:
This should just be `const { plugin } = this.props` I think (cf https://github.com/Automattic/wp-calypso/blob/9fa4f44304e7c5d04edbd4a73b46dda6b9a1a3dd/client/my-sites/plugins/plugin-meta/index.jsx#L473-L480).